Extensions/AnchorBehavior/anchorruntimebehavior.ts

```diff
@@ -270,8 +270,13 @@ namespace gdjs {
             this._rightEdgeAnchor !== HorizontalAnchor.None &&
             this._leftEdgeAnchor !== HorizontalAnchor.None
           ) {
-            this.owner.setWidth(right - left);
-            this.owner.setX(left);
+            const width = right - left;
+            this.owner.setX(
+              left +
+                ((this.owner.getX() - this.owner.getDrawableX()) * width) /
+                  this.owner.getWidth()
+            );
```

Comment:
Should we had `if (this.owner.getX() === this.owner.getDrawableX())` to avoid extra computations 99% of the time at the cost of the `if`?